Automattic/jetpack
# Sync: add FSE toggle option

See https://github.com/Automattic/wp-calypso/issues/54469

#### Changes proposed in this Pull Request:
Adds `wpcom_is_fse_activated` to sync list

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
This will be set by our own plugin (Editing Toolkit) so shouldn't be an issue based on previous PRs

#### Testing instructions:
1. Run this PR on a Jetpack site
2. Patch your sandbox with D64660-code
3. Point your Jetpack site to your sandbox: PCYsg-efV-p2#making-the-jetpack-plugin-believe-that-public-api-wordpress-com-lives-in-your-sandbox
2. Run the latest Editing Toolkit plugin on the Jetpack site
3. Activate an FSE theme (TT1 Blocks and Q are findable on dotorg)
4. Go to Appearance → Site Editor (beta) and turn on the Site Editor
5. On your dotcom sandbox, run `wp --url=YOUR_JETPACK_DOMAIN option get wpcom_is_fse_activated`, which should return `1`.
